src/app/blog/blog.routes.ts

```
@@ -0,0 +1,48 @@
+import {any, z} from 'zod';
+
+// Define Zod schema for validating route parameters
+const idParamSchema = z.object({
+    id: z.string().uuid("Invalid ID format"), // Validate that the ID is a valid UUID
+});
+
+const router = express.Router();
+
+// Middleware for validating params
+const validateParamsMiddleware = (schema: z.ZodSchema) => {
+    return (req: Request, res: Response, next:Function): void => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)